canonicalwebteam/store_api/stores/charmstore.py

```diff
@@ -339,7 +339,7 @@ def revoke_invites(self, publisher_auth, name, emails):
             headers=self._get_authorization_header(publisher_auth),
             json=payload,
         )
-        return self.process_response(response)
+        return response
```

Comment:
has the response already been processed at this point?

Reply:
I believe so. It looks like the response we get from the API is already processed.

Reply:
It's not necessarily different. The majority of the `post` requests in that file are just returning the response rather than processing it.

Reply:
You should reopen this PR, the exception thrown was because the response body above is empty, so `process_response` returns a JSOndecode error.